R/imports.R

```diff
@@ -0,0 +1,2 @@
+#' @importFrom utils data
+NULL
```

Comment:
Do something like this for each function from an external package that you want to use here; but you can avoid this using the syntax `packagename::functionname()`.